numpy/core/src/common/ufunc_override.c

```diff
@@ -93,8 +93,16 @@ PyUFuncOverride_GetOutObjects(PyObject *kwds, PyObject **out_kwd_obj, PyObject *
         return 0;
     }
     if (PyTuple_CheckExact(*out_kwd_obj)) {
-        *out_objs = PySequence_Fast_ITEMS(*out_kwd_obj);
-        return PySequence_Fast_GET_SIZE(*out_kwd_obj);
+        PyObject *seq = PySequence_Fast(*out_kwd_obj, "cannot convert");
+        int ret;
+        if (seq == NULL) {
+            return -1;
+        }
+        Py_DECREF(*out_kwd_obj);
```

Comment:
Can use `Py_SETREF(*out_kwd_obj, seq)` here

Reply:
Is this now formally supported? I don't see it in https://docs.python.org/3.7/c-api/refcounting.html

Reply:
More relevantly though, I added a backport of it, in `npy_compat` I beleive